libraries/AP_VideoTX/AP_SmartAudio.cpp

```diff
@@ -254,6 +254,7 @@ void AP_SmartAudio::send_request(const Frame& requestFrame, uint8_t size)
     const uint8_t *request = reinterpret_cast<const uint8_t*>(&requestFrame);
 
     // write request
+    _port->write((uint8_t)0x00); //pull the port down as per specifications.
```

Comment:
This needs to be done based on an option bit in VTX_OPTIONS. When I was testing originally this interfered with TBS devices, and TBS devices are the main supported devices.

Reply:
That would be an actual breakage of their own protocol: 
![image](https://user-images.githubusercontent.com/20860004/120377507-82271780-c31d-11eb-90b6-b2b087964443.png)
Am I wrong?